README.md

```diff
@@ -113,6 +113,40 @@ juju remove-relation zookeeper-k8s tls-certificates-operator
 
 Note: The TLS settings here are for self-signed-certificates which are not recommended for production clusters, the `tls-certificates-operator` charm offers a variety of configurations, read more on the TLS charm [here](https://charmhub.io/tls-certificates-operator)
 
+## Monitoring
+
+The Charmed ZooKeeper K8s Operator comes with several exporters by default. The metrics can be queried by accessing the following endpoints:
+
+- JMX exporter: `http://<pod-ip>:9998/metrics`
+- ZooKeeper metrics: `http://<pod-ip>:7000/metrics`
+
+Additionally, the charm provides integration with the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack).
+
+Deploy `cos-lite` bundle in a Kubernetes environment. This can be done by following the [deployment tutorial](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s). It is needed to offer the endpoints of the COS relations. The [offers-overlay](https://github.com/canonical/cos-lite-bundle/blob/main/overlays/offers-overlay.yaml) can be used, and this step is shown on the COS tutorial.
+
+Once COS is deployed, we can find the offers from the zookeeper model:
+```shell
+# We are on the `cos` model. Switch to `zookeeper` model
+juju switch <zookeeper_model_name>
+juju find-offers <k8s_controller_name>:
+```
+
+A similar output should appear, if `micro` is the k8s controller name and `cos` the model where `cos-lite` has been deployed:
+```
+Store  URL                   Access  Interfaces                         
+micro  admin/cos.grafana     admin   grafana_dashboard:grafana-dashboard
+micro  admin/cos.prometheus  admin   prometheus_scrape:metrics-endpoint
+. . .
+```
+
+Now, integrate zookeeper with the `metrics-endpoint`, `grafana-dashboard` and `logging` relations:
+```shell
+juju relate micro:admin/cos.prometheus zookeeper-k8s
+juju relate micro:admin/cos.grafana zookeeper-k8s
+juju relate micro:admin/cos.loki zookeeper-k8s
+```
```

Comment:
**question:** Given that they're both in K8s, do we need to `offer` the relations? Could we advise to relate directly?

Reply:
@zmraul I believe this is more for your part

Reply:
Yes we still need the offer, as ideally you would be deploying cos in a different model. The controller part could be omitted I believe, but it doesn't hurt to be explicit here.